### PR TITLE
Update RELEASE_PROCESS.md

### PR DESCRIPTION
**Pull request by lbakerchef**
Update RELEASE_PROCESS.md

Signed-off-by: Lincoln Baker <lbaker@chef.io>

### CODE REVIEW

File: RELEASE_PROCESS.md [hunk 1]
```diff
@@ -160,17 +163,23 @@ Note that no changes other than CHANGELOG/RELEASE_NOTES changes should land on m
 Chef / [chef/chef-server:main] habitat/build / main
 https://buildkite.com/chef/chef-chef-server-main-habitat-build
 
+==============
+This section (below) needs additional information, such as... what are we even in, a dev env, a terraform machine?  Which scenarios, configs, platforms?  How to run the supermarket test?  If it's a terraform scenario, how to activate the supermarket test when running it (I looked in the docs but didn't see anything)?  What about a step by step walkthrough as we have in sections above?
+==============
```
Comment (PrajaktaPurohit) [inline RELEASE_PROCESS.md:168]: https://github.com/chef/umbrella/tree/main/chef-server/scenarios/aws/supermarket-upgrade is run for every test cycle. So we do not need manual supermarket testing anymore.

File: RELEASE_PROCESS.md [hunk 2]
```diff
@@ -240,24 +249,26 @@ RELEASE CHECKLIST
...
     AZURE
-        ipv4                                PENDING [credentials issue]
-        ipv6                                PENDING [credentials issue]
-- special testing                           PASS
-- check that hab builds are successful      PASS
-- verify that manifest contains correct     FAIL
+        ipv4                                TERRAFORM ERROR
+        ipv6                                TERRAFORM ERROR
+- check that hab builds are successful      PASSED
+- supermarket                               SUPPORT REQUIRED
```
Comment (PrajaktaPurohit) [inline RELEASE_PROCESS.md:267]: We can remove this from the manual testing list now.

File: RELEASE_PROCESS.md [hunk 3]
```diff
@@ -240,24 +249,26 @@ RELEASE CHECKLIST
...
+- automate                                  SUPPORT REQUIRED
+- special testing                           PENDING
```
Comment (PrajaktaPurohit) [inline RELEASE_PROCESS.md:269]: Would this be specific to the release?

File: RELEASE_PROCESS.md [hunk 4]
```diff
@@ -240,24 +249,26 @@ RELEASE CHECKLIST
...
+- verify that manifest contains correct     PASSED
```
Comment (PrajaktaPurohit) [inline RELEASE_PROCESS.md:270]: could you add the link for where the manifest will be found?

File: RELEASE_PROCESS.md (hunk 1 above)
Comment (lbakerchef) [reply RELEASE_PROCESS.md:168]: @PrajaktaPurohit Should I remove this section from the docs?

File: RELEASE_PROCESS.md (hunk 2 above)
Comment (lbakerchef) [reply RELEASE_PROCESS.md:267]: Roger that.

File: RELEASE_PROCESS.md (hunk 3 above)
Comment (lbakerchef) [reply RELEASE_PROCESS.md:269]: Yes.  I'll clarify.

Comment (lbakerchef) [reply RELEASE_PROCESS.md:269]: See "Special Testing" section.

File: RELEASE_PROCESS.md (hunk 4 above)
Comment (lbakerchef) [reply RELEASE_PROCESS.md:270]: It's in the appropriate section above.

File: RELEASE_PROCESS.md [hunk 5]
```diff
@@ -77,6 +77,9 @@ Umbrella Testing Step-by-Step:
 1. Optionally, fill-in the 'Message' field with something descriptive.
 1. Select 'Create Build'.
 
+============================================
+does manage testing need changes, or deletion?
+============================================
```
Comment (lbakerchef) [inline RELEASE_PROCESS.md:82]: ...since you said the bug might have been fixed (@PrajaktaPurohit)?

File: RELEASE_PROCESS.md (hunk 1 above)
Comment (PrajaktaPurohit) [reply RELEASE_PROCESS.md:168]: yes!

Comment (lbakerchef) [reply RELEASE_PROCESS.md:168]: Done.

File: RELEASE_PROCESS.md [hunk 6]
```diff
@@ -77,6 +77,9 @@ Umbrella Testing Step-by-Step:
 1. Optionally, fill-in the 'Message' field with something descriptive.
 1. Select 'Create Build'.
 
+============================================
+does manage testing (below) need changes, or deletion?
```
Comment (PrajaktaPurohit) [inline RELEASE_PROCESS.md:81]: this is still true.

**Review by PrajaktaPurohit — APPROVED** [review]

**Pull request merged**